### PR TITLE
Add source code pro font to the distributed package

### PR DESCRIPTION
**Pull request by andrewgiessel**
This uses package_data, as discussed [here](https://setuptools.pypa.io/en/latest/userguide/datafiles.html).

Also fixes one small typo.

### CODE REVIEW

**Comment by andrewgiessel** [comment]
@ericmjl  I'm not sure the best way to test this- any thoughts?

File: setup.py [hunk 1]
```diff
@@ -126,6 +126,7 @@ def download_mafft(kind="deb") -> Path:
         "multipledispatch",
         "python-codon-tables",
     ],
+    package_data={"seqlike": ["*.ttf"]},
```
Comment (ericmjl) [inline setup.py:129]: Note for myself: I think this is the fix. However, not 100% sure. Will need to pull down to check.

Comment (ericmjl) [reply setup.py:129]: Confirming the presence of `FreeMono.ttf` in the pip-installation location:

```
(base) root@codespaces_81c95a:/opt/conda/lib/python3.9/site-packages/seqlike# ls
AlignCommandline.py  SeqLike.py          __init__.py  alignment_commands.py  alphabets.py     draw_utils.py  utils
FreeMono.ttf         SeqLikeAccessor.py  __pycache__  alignment_utils.py     codon_tables.py  encoders.py
```

Comment (ericmjl) [reply setup.py:129]: Leaving another note for myself:

1. We can programmatically execute `python -m build`.
2. We can then unzip and untar the resulting `.tar.gz` file in `dist/`.
3. We can then assert that `FreeMono.ttf` exists in the distributed source dir.

**Comment by ericmjl** [comment]
@andrewgiessel I'm going to merge and cut a new patch release. It's good to go now.

**Pull request merged**